src/components/Home/index.jsx

```diff
@@ -1,8 +1,517 @@
+`;
+
+const Desc = Styled.div`
+  font-size: 1.5rem;
+`;
+
+const Logo = Styled.img`
+  width: 15vw;
+  height: auto;
+  min-width: 120px;
+  top: 20px;
+  left: 35px;
+  position: absolute;
+  @media(max-width: 600px){
+    top: 15px;
+    left: 15px;
+  }
+  z-index: 10;
+`;
+
+const GroundContent = Styled.div`
+  width: 100%;
+  z-index: 3;
+  @media(max-width: 570px){
+    margin-top: -60px;
+  }
+  @media(max-width: 380px){
+    margin-top: -90px;
+  }
+`;
+
+const BackgroundRoad = Styled.img`
+  width: 101%;
+  margin-left: -2px;
+`;
+
+const ForegroundBush = Styled.img`
+  position: relative;
+  z-index: 5;
+  width: 90%;
+  margin: -2.5% 5%;
+`;
+
+const Car = Styled.img.attrs(props => {
+  let p = props.position - 80;
+  return({
+    style: {
+      transform: 'translate(' + (p > 0 ? p * 5 : 0) + 'vw)'
+    }
+  })
+})`
+  z-index: 2
+  width: 10%;
+  margin: -3.8% 0;
+`;
+
+const RoadWrapper = Styled.div`
+  z-index: 2
+  width: 100vw;
+  margin-top: -30vw;
+  min-width: 1000px;
+  overflow: hidden;
+`;
+
+const TimeWrapper = Styled.div`
+  overflow: hidden;
+  display: flex;
+  flex-direction: row;
+  font-size: 24px;
+  justify-content: space-around;
+  color: white;
+  margin-top: -10px;
+  padding: 50px 0 25px 0;
+  background: #4B8655;
+`;
+
+const Clickable = Styled.div`
+  letter-spacing: 0.1em;
+  font-family: Montserrat;
+  font-style: normal;
+  font-size: 1.2em;
+  letter-spacing: 0.1em;
+  &:hover{
+    cursor: pointer;
+  }
+`;
+
+const FAQContainer = Styled.div`
+  z-index: 6;
+  overflow: hidden;
+  background: #4B8655;
+  padding-bottom: 9vw;
+  margin-top: -25px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  color: white;
+
+  @media(max-width: 3000px){
+    padding-bottom: 7vw;
+  }
+
+  @media(max-width: 2500px){
+    padding-bottom: 5vw;
+  }
+
+  @media(max-width: 2500px){
+    padding: 40px;
+  }
+  
+  @media(max-width: 900px){
+    grid-template-columns: 1fr;
+  }
+  @media(max-width: 650px){
+    padding: 40px 0;
+  }
+`;
+
+const FAQTitle = Styled.div`
+  padding: 10px;
+  font-size: 16px;
+`;
 
 export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      FAQState: 'General',
+      scrollPos: 0,
+    };
+
+    this.handleScroll = this.handleScroll.bind(this);
+    this.changeFAQ = this.changeFAQ.bind(this);
+  }
+
+  componentDidMount() {
+    window.addEventListener('scroll', this.handleScroll, true);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.handleScroll);
+  }
+
+  changeFAQ(title) {
+    this.setState({FAQState: title});
+  }
+
+  handleScroll() {
+    let h = document.body.clientHeight;
+    let st = document.body.scrollTop;
+    let sh = document.body.scrollHeight;
+    // console.log(h, st, sh);
+    this.setState({scrollPos: st / (sh - h) * 100});
+  }
+
   render() {
-    return <p>/home</p>;
+    const { FAQState, scrollPos }= this.state;
+
+    const Descriptions = [
```

Comment:
constants should be UPPER_SNAKE_CASE

Reply:
I really like how you grouped all the text into objects here!! You might want to put all of these constants into a separate JS file though. They account for a lot of lines and detract from the CSS/JSX